downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -48,23 +48,22 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 11 (2201.11.0)
 
 ## Ballerina library updates
 
-### Features
+### New features
 
 #### `http` package
 
-- Added static code rules 
-- Added relax data binding support for service and client data binding
+- Added relaxed binding support for service and client data binding.
```

Comment:
```suggestion
- Added relaxed binding support for service and client data binding. This provides the flexibility to assign `nil` values to optional fields and absent values to nilable fields in the data projection.
  ```ballerina
  // Enable relaxed data binding in the client side
  http:Client httpClient = check new("http://localhost:9090", laxDataBinding = true);

  // Enable relaxed data binding in the server side
  @http:ServiceConfig {laxDataBinding: true}
  service /api on new http:Listener(9090) {
  }
  ```
```

Reply:
````suggestion
- Added relaxed binding support for service and client data binding. This provides the flexibility to bind nil values to optional fields and absent values to nilable fields.

  ```ballerina
  // Enable relaxed data binding on the client side.
  http:Client httpClient = check new("http://localhost:9090", laxDataBinding = true);
  
  // Enable relaxed data binding on the server side.
  @http:ServiceConfig {laxDataBinding: true}
  service /api on new http:Listener(9090) {
  }
  ```
````